library/object/config/config.php

```diff
@@ -181,15 +182,6 @@ public function append($options)
                         else $this->set($key, $value);
                     }
                 }
-                else
-                {
-                    foreach($options as $value)
-                    {
-                        if (!in_array($value, $this->__options, true)) {
```

Comment:
@oligriffiths By removing this check you are potentially adding the same value multiple times. This can be negative side-effects. Can you review this ?